config/locales/mailers/en.yml

```diff
@@ -60,3 +60,12 @@ en:
         para_2: "Again, great job! We look forward to working with you again in the future."
         sign_off: "Sincerely,"
         from: "The 18F Micro-purchase team"
+      auction_rejected:
+        subject: "18F Micro-purchase Auction Status"
+        para_1: >-
+          We regret to inform you that your work for %{auction_title} was rejected
```

Comment:
This message suggests that we will provide a reason for rejection on the auction. Is that true?